strava_api_v3/api/athletes_api.py

```diff
@@ -38,17 +38,17 @@ def get_logged_in_athlete(self, **kwargs):  # noqa: E501
 
         Returns the currently authenticated athlete.  # noqa: E501
         This method makes a synchronous HTTP request by default. To make an
-        asynchronous HTTP request, please pass async=True
```

Comment:
Looks like the rpelace all was a bit aggressive. I think you mean this line to be 
`asynchronous HTTP request, please pass async_req=True` and there are similar errors in the rest of the file.

Reply:
I don't undersant what you mean with that, could you indicate me what i should change? 

It's my first package request, thanks!!!!!!